Brain_I2C/Brain_I2C.ino

```diff
@@ -45,10 +50,50 @@ void loop() {
       index++;
     }
     for (int i=0; i<6; i++) {
-      Serial.print(farrbconvert.sensorReadings[i]);
-      Serial.print(" ");
+      float reading = farrbconvert.sensorReadings[i];
+      if (reading > 0) {
+      }
+    }
+    String packet = make_packet(sensor);
+    Serial.println(packet);
+    RFSerial.println(packet);
+    delay(sensor.clock_freq);
```

Comment:
With this, what will happen is that we'll be running at the speed of the slowest sensor since we'll be delaying by that amount each time. In this case, we'll check pressure and wait for 1 sec, and then check temp and wait for half a sec, so we still check pressure once for each time we check pressure.

What we want, is to check temp twice for every time we check pressure. So I think you should always delay by 100 or something reasonably small. Instead of clock_freq being a time to delay, make it a number denoting how often to check, for example 2 means every other, 5 means every 5th time. 
Add a loop counter that counts the number of iterations loop goes through, and if that counter % curr_freq == 0, then do the reading. else, do nothing. 
Every so often you can reset that counter to 0, probably choosing the largest number we have for curr_freq as the point at which to return to 0.